src/engine/game/game.lua

```diff
@@ -542,6 +542,26 @@ function Game:getSoulPartyMember()
     return current
 end
 
+function Game:setGameTitle(title)
+    love.window.setTitle(title)
+end
+
+function Game:getGameTitle()
+    return love.window.getTitle()
+end
+
+function Game:setGameIcon(iconData)
+    -- Make sure our icon isn't null.
+    if iconData == nil then return end
```

Comment:
i dont know if this is required, if someone passes bad data then it _should_ error